plugins/base/base.go

```diff
@@ -0,0 +1,35 @@
+package shared
```

Comment:
Did you intend to put these in the `github.com/hashicorp/nomad/plugins/base` package and have the package named `shared`?

Reply:
Nope! Was a refactoring accident.